src/main/java/com/dumbdogdiner/stickyapi/bukkit/plugin/StickyPlugin.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.plugin;
+
+import com.dumbdogdiner.stickyapi.common.translation.LocaleProvider;
+import lombok.Getter;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public abstract class StickyPlugin extends JavaPlugin {
```

Comment:
This class seems a lot less useful than it should be. Perhaps this should be an interface instead?

Reply:
^